### ISSUE TITLE
Examples do not return

### ISSUE THREAD

**Issue by desowin** [Shouqun/node-dbus#170]
Examples that call methods (eg. hello.js, get_property.js) do not return after finishing. I think there shouldn't be any requests left after the called methods succeed/fail and hence node should stop the process.

**Comment by bryanburgers** [comment]
Yeah, I'm guessing the examples probably need a `.disconnect()` somewhere. Shouldn't be a big deal. How would you feel about giving it a try with a PR? You take a first pass at it, and if you decide it's not for you, somebody else can have a look?

**Comment by desowin** [comment]
It indeed is `bus.disconnect()` that needs to be called after bug connection is no longer needed. In hello.js that's pretty easy, as it can be added to callback passed to Hello function call. However in get_property.js there are multiple calls (getProperty and getProperties). What would be the correct way to handle that there? Would a simple "callbacks yet to be called" counter suffice?
